util/appcontext/appcontext.go

```diff
@@ -1,18 +1,22 @@
-package main
+package appcontext
 
 import (
 	"context"
 	"os"
 	"os/signal"
 	"sync"
 
+	"github.com/Sirupsen/logrus"
 	"golang.org/x/sys/unix"
 )
 
+// appcontext is a static context that reacts to termination signals of the
```

Comment:
This comment is supposed to be generally about the package. I'll move it to the `Context()` function.